CMSIS/AmbiqMicro/Include/apollo3.h

```diff
@@ -126,8 +129,18 @@ typedef enum {
 #define __CM4_REV                 0x0100U       /*!< CM4 Core Revision                                                         */
 #define __NVIC_PRIO_BITS               3        /*!< Number of Bits used for Priority Levels                                   */
 #define __Vendor_SysTickConfig         0        /*!< Set to 1 if different SysTick Config is used                              */
+#define __VTOR_PRESENT                 1        /*!< Set to 1 if CPU supports Vector Table Offset Register                     */
 #define __MPU_PRESENT                  1        /*!< MPU present                                                               */
 #define __FPU_PRESENT                  1        /*!< FPU present                                                               */
+#define __FPU_DP                       0        /*!< Double Precision FPU                                                      */
+#define __DSP_PRESENT                  0        /*!< DSP extension present                                                     */
+#define __ICACHE_PRESENT               1        /*!< Instruction Cache present                                                 */
```

Comment:
Enable ICACHE_PRESENT and DCACHE_PRESENT will cause compiling issue. See the comment on apollo4p.h.